slack_sdk/models/blocks/block_elements.py

```diff
@@ -1035,7 +1308,6 @@ def __init__(
         *,
         action_id: Optional[str] = None,
         placeholder: Optional[Union[str, dict, TextObject]] = None,
-        confirm: Optional[Union[dict, ConfirmObject]] = None,
```

Comment:
Input block element does not accept confirm object. This has been wrong for a long time.